ruff_dev/src/generate_rules_table.rs

```diff
@@ -47,24 +47,33 @@ pub fn main(cli: &Cli) -> Result<()> {
     for origin in RuleOrigin::iter() {
         let prefixes = origin.prefixes();
         let codes_csv: String = prefixes.as_list(", ");
-        table_out.push_str(&format!("### {} ({codes_csv})", origin.title()));
+        table_out.push_str(&format!("### {} ({codes_csv})", origin.name()));
         table_out.push('\n');
         table_out.push('\n');
 
         toc_out.push_str(&format!(
             "   1. [{} ({})](#{}-{})\n",
-            origin.title(),
+            origin.name(),
             codes_csv,
-            origin.title().to_lowercase().replace(' ', "-"),
+            origin.name().to_lowercase().replace(' ', "-"),
             codes_csv.to_lowercase().replace(',', "-").replace(' ', "")
         ));
 
-        if let Some((url, platform)) = origin.url() {
+        if let Some(url) = origin.url() {
+            let host = url
+                .trim_start_matches("https://")
+                .split('/')
+                .next()
+                .unwrap();
             table_out.push_str(&format!(
                 "For more, see [{}]({}) on {}.",
-                origin.title(),
+                origin.name(),
                 url,
-                platform
+                match host {
+                    "pypi.org" => "PyPI",
+                    "github.com" => "GitHub",
+                    host => host,
```

Comment:
Similarly, let's error on this branch for now.

Reply:
Done.